src/components/v2/appDetails/k8Resource/nodeType/Node.component.tsx

```diff
@@ -262,7 +262,7 @@ function NodeComponent({ handleFocusTabs, externalLinks, monitoringTools, isDevt
                             )}
                         </div>
                     )}
-                    <div className="node-row m-0 resource-row">
+                    <div className="node-row m-0 resource-row" data-testid="">
```

Comment:
Remove the empty attribute if not in use

Reply:
Done